cfpb/design-stories
# Add number options to Icon component

Adds numbers to the `Icon` component. Number icons exist in the [icons directory](https://github.com/cfpb/design-system/tree/main/packages/cfpb-icons/src/icons) but they're not mentioned on the DS website or used in cf.gov.

## Changes

- Adds number icons

## How to test this PR

1. PR tests should pass.
2. Visit the PR preview and you should see numbers in the Icon section.
